src/components/AdminV2/cards/NumberCard/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Icon, Card, Collapsible,
+} from '@openedx/paragon';
+
+import DetailsAction from './DetailsAction';
+
+const NumberCard = ({
+  title, icon, description, detailActions,
+}) => {
+  const handleDetailsActionClick = (event) => {
+    const element = document.getElementById('learner-progress-report');
+    if (event) {
+      event.target.click();
+    }
+    if (element) {
+      setTimeout(() => {
+        element.scrollIntoView({ behavior: 'smooth' });
+      }, 0);
```

Comment:
Could we try [requestAnimationFrame](https://phuoc.ng/collection/this-vs-that/set-timeout-zero-vs-request-animation-frame/#requestanimationframe) instead of `setTimeout`